css/style.css

```
@@ -0,0 +1,331 @@
+/* General Css */
+/* Google font */
+@import url('https://fonts.googleapis.com/css?family=Muli:400,600,700,800');
+
+/* UPDATE BY ZUBAIR */
+* {
+    font-family: 'Muli', sans-serif;
+    color: #56576E;
+    line-height: 1.6em;
+    font-weight: 400;
+    font-size: 16px;
+}
+
+h1, h2, h3, h4, h5, h6{
+    margin: 0 0 15px;
+    font-weight: 700;
+    line-height: 1.4em;
+    color: #22233b;
+}
+
+a {
+    text-decoration: none;
+    outline: none;
+    -webkit-transition: 0.4s;
+    -o-transition: 0.4s;
+    transition: 0.4s;
+    color: #56576e;
+}
+
+input:focus,
+button:focus,
+a:focus,
+a:hover {
+    text-decoration: none;
+    outline: none;
+    color: #6858db;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+/* END UPDATE BY ZUBAIR */
+/* General Css */
+
+/* START NAV AND BANNER FEAT#12, BUILD BY ZUBAIR */
+/* start nav */
+.nav-header{
+    padding: 15px 0 15px 0;
+    background: white;
+}
+
+.nav-item{
+    padding:10px;
+}
+
+.home{
+    color: #6858db;
+}
+
+.nav-item a{
+    font-size: 16px;
+    font-weight: 600;
+    padding: 20px;
+}
+
```

Collapsed menu nav-links have wrong color.
```suggestion
.nav-item a:focus{
  color: #6858db;
}
```

---

@marcin-wicha I don't understand. what is the right color? can you help?

---

@a2-zubair Sure. the code sample I sent, fixed this. The links when you hover on them should be this purple color, not standard blue.